src/main/java/org/teamresistance/frc/Snorfle.java

```diff
@@ -0,0 +1,34 @@
+package org.teamresistance.frc;
+
+import org.strongback.command.Command;
+import org.strongback.components.Motor;
+
+/**
+ * @author Shreya Ravi
+ */
+public class Snorfle extends Command {
+  Motor rollers;
+  Motor spinners;
+  double rollerSpeed;
+  double spinnerSpeed;
+
+  public Snorfle(Motor rollers, Motor spinners, double rollerSpeed, double spinnerSpeed) {
```

Comment:
You probably want this constructor to be `protected` to prevent it from coming up as an option when instantiating any of its subclasses.